custom_model_runner/datarobot_drum/drum/common.py

```diff
@@ -306,6 +306,7 @@ class ModelMetadataKeys(object):
     # customPredictor section is not used by DRUM,
     # it is a place holder if user wants to add some fields and read them on his own
     CUSTOM_PREDICTOR = "customPredictor"
+    PARAMETERS = "parameters"
```

Comment:
not hyperparameters?

Reply:
changed to hyperparameters (its a single word so no camelcase on the 'p')